programs/drift/src/state/user.rs

```diff
@@ -1524,6 +1524,26 @@ impl Order {
             || (self.triggered()
                 && !(self.reduce_only && self.is_bit_flag_set(OrderBitFlag::NewTriggerReduceOnly)))
     }
+
+    pub fn is_low_risk_for_amm(
+        &self,
+        mm_oracle_delay: i64,
+        clock_slot: u64,
+        is_liquidation: bool,
+    ) -> DriftResult<bool> {
+        if self.market_type == MarketType::Spot {
+            return Ok(false);
+        }
+
+        let order_older_than_oracle_delay = {
```

Comment:
i feel like we may still want the order to have been onchain longer than the speed bump?

e.g. the the taker speedbump is 5, the oracle delay could be 1 and we'd let the order fill after 2 slots. is that ok?